packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -591,6 +591,19 @@ class DedicatedCloudService {
     });
     return filterRequest;
   }
+
+  grantActiveDirectory(serviceName, activeDirectoryId, params, type) {
+    return this.$http
+      .post(
+        `/dedicatedCloud/${serviceName}/federation/activeDirectory/${activeDirectoryId}/${
+          type === 'user'
+            ? 'grantActiveDirectoryUser'
+            : 'grantActiveDirectoryGroup'
```

Comment:
i guess it's better to have this condition where the call is made.

Reply:
It's applied.